twweet_cli.py

```diff
@@ -47,7 +47,7 @@ def get_all_tweets(screen_name):
         # update the id of the oldest tweet less one
         oldest = alltweets[-1].id - 1
 
-        print "...%s tweets downloaded so far"%(len(alltweets))
+        print("...%s tweets downloaded so far".format(len(alltweets)))
```

Comment:
And in line 99, `with open('%s.csv'%hash_tag, 'wb') as f:` --> `with open('%s.csv'%hash_tag, 'w') as f:`

And in line 56, `with open('%s_tweets.csv' % screen_name,'wb') as f:` --> `with open('%s_tweets.csv' % screen_name,'w') as f:`